### DISCUSSION TITLE
shopt: globstar: invalid shell option name

### DISCUSSION THREAD

**Discussion by saschwarz** [evertramos/nginx-proxy-automation#344] [Q&A]
I'm following the upgrade instructions and I get this error on OSX. I tried using `/bin/sh` and also get the same error:

```
> ./fresh-start.sh                                                                                           
./fresh-start.sh: line 17: shopt: globstar: invalid shell option name
readlink: illegal option -- f
usage: readlink [-n] [file ...]
\U26a1 Start execution './fresh-start.sh Password:
```

**Comment by evertramos** [comment]
Hello @saschwarz!

Thanks for the discussion.

I use linux and bash, so might be some options in OSX shell that does not exist such as the _shopt_.

Please comment this line below with **"#"**.

https://github.com/evertramos/nginx-proxy-automation/blob/578ea0b94ccc3a78c13d6c3e983d8e6e3fc31694/bin/fresh-start.sh#L17

Let me know how it goes.
